frontend/storybook/src/components/entry/homepage/whatsnew-paragraph.ts

```diff
@@ -0,0 +1,60 @@
+import "@elements/entry/home/TOSORT/icon-wparagraph";
+import "@elements/entry/home/sections/whatsnew-section";
+import "@elements/core/titles/variants/title-section";
+import "@elements/core/titles/variants/subtitle";
+import "@elements/entry/home/TOSORT/list-type";
+import "@elements/core/buttons/rectangle";
+import "@elements/core/dividers/circle-div";
+import "@elements/entry/home/TOSORT/content-wimg";
+
+
+  
+export default {
+  title: 'Homepage',
+}
+
+
+
+
+const STR_TITLE = "What’s new?";
+const STR_SUBTITLE="HOP TV - New Hebrew Series";
+const STR_WHITE = "white";
+const STR_DARKBLUE = "darkblue";
+ const STR_LARGE = "large";
+ const STR_PINK = "pink";
```

Comment:
icon paths and color properties should not be in STR_ prefixes. The intent of the STR_ prefixes is that we will eventually move them out of code completely and they will be set in a database for copywriters/translators.

For story mock data, just set the default properties in DEFAULT_ARGS directly (and other radio options in `argTypes`)